transport/conn.go

```diff
@@ -208,6 +217,10 @@ func (c *connReader) recv() response {
 		return r
 	}
 	r.Response = c.parse(r.Header.OpCode)
+	if r.Response == nil {
+		r.Err = fmt.Errorf("response type not supported")
```

Comment:
In `func (c *connReader) parse(op frame.OpCode) frame.Response` we `log.Fatalf` when response type is not supported so I think our program will never visit this if statement?

Reply:
I added this because of certain responses like Result or Event. For example, we get response with correct header describing some event. Event starts with a string describing the type of event, but if it is not supported (string is incorrect), we return nil from parse event. That's how we can still return nil from parse.
Maybe we should log.Fatalf inside parsing event, but it seems like an overkill.

Reply:
You're right, that is correct, thanks for the explanation!